pkg/apis/apps/v1/defaults_test.go

```diff
@@ -287,7 +290,8 @@ func TestSetDefaultStatefulSet(t *testing.T) {
 					Template: defaultTemplate,
 					UpdateStrategy: appsv1.StatefulSetUpdateStrategy{
 						RollingUpdate: &appsv1.RollingUpdateStatefulSetStrategy{
-							Partition: &notTheDefaultPartition,
+							Partition:      &notTheDefaultPartition,
+							MaxUnavailable: &notTheDefaultMaxUnavailable,
```

Comment:
Why is this in this test case?

Reply:
i will remove it. Earlier i was testing with flag always enabled and hence it was added